MSON Specification.md

```diff
@@ -439,12 +439,13 @@ A _Wildcard Type Name_ MAY only be used in a _[Type Specification][]_ for _[Memb
 #### 3.5.3 Type Attribute
 Defines extra attributes associated with the implementation of a type.
 
-- `required` - instance of this type is required
-- `optional` - instance of this type is optional (default)
-- `fixed`    - instance of this type structure and values are fixed
-- `nullable` - instance of this type *Value* MAY be unset (e.g. `null` or `nil`). `nullable` may only be used within properties of objects.
-- `sample`   - Alternate way to indicate a _[Value][]_ is a sample. See _[Sample][]_.
-- `default`  - Alternate way to indicate a _[Value][]_ is a default. See _[Default][]_.
+- `required`   - instance of this type is required
+- `optional`   - instance of this type is optional (default)
+- `fixed`      - instance of this type structure and values are fixed. This attribute is inheritable: all internal structures became fixed too.
+- `fixed-type` - instance of this type structure is fixed, value - is not. This attribute is unheritable.
```

Comment:
Are you trying to say that the `fixed-type` is not propagating to nested member types? If, so what is the reasoning for this. Also I'd suggest to use a different expression (but unheritable) to avoid any misinterpretation..